### ISSUE TITLE
Binding multiple actions to one key shows them unbound in the UI

### ISSUE THREAD

**Issue by ChillerDragon** [ddnet/ddnet#6905]
```
unbindall
bind a "+left;+hook"
```

shows hooking and walking left as not set in the ui

and setting it overwrites the whole bind

I always considered that a feature to know which controls I should not set in the UI to not loose more complex binds. But I guess technically that is a bug. And it would be nice to not hide the binds and maybe even indicate that the key has multiple binds on it. Or even allow merging binds over the ui.

**Comment by def-** [comment]
I consider this correct behavior.

**Comment by ChillerDragon** [comment]
> I consider this correct behavior.

You consider walking left being shown as unbound in the ui eventho pressing the key a results in walking left?

**Comment by heinrich5991** [comment]
The general case is probably too annoying to implement. I think it's okay if the menu only shows keys that are bound to exactly one action.

**Comment by ChillerDragon** [comment]
> The general case is probably too annoying to implement. I think it's okay if the menu only shows keys that are bound to exactly one action.

Yea I can also see a proper implementation to be quite complex. But we have the super motivated robstar now! Maybe he wants to give it a try :)

**Comment by heinrich5991** [comment]
I don't think this should be implemented before we have a concrete UI design. As it stands, this issue is probably too vague to be implemented.

**Comment by sjrc6** [comment]
it would be nice to at least distinguish between unbound and multiple bound, like showing *** if multiple keys are bound.
